components/brave_wallet_ui/components/desktop/views/accounts/account.tsx

```diff
@@ -176,11 +179,23 @@ export const Account = () => {
     return list
   }, [userVisibleTokensInfo, selectedAccount, networkList])
 
+  const {
+    data: tokenBalancesRegistry,
+  } = useBalancesFetcher(selectedAccount && networkList
+    ? {
+        accounts: [selectedAccount],
+        networks: networkList
```

Comment:
`useBalancesFetcher()` does not accept a tokens list, as it's designed to scan all tokens (fungible or otherwise). Using the right tokens list is already taken care of by the hook.

This code can probably be optimised further, but prefer doing that in another PR if we feel the need.